apache2.yaml

```diff
@@ -0,0 +1,221 @@
+        --with-suexec-logfile=/var/log/apache2/suexec.log \
+        --with-suexec-bin=/usr/sbin/suexec \
+        --with-suexec-uidmin=99 \
+        --with-suexec-gidmin=99 \
+        --with-apr=/usr/bin/apr-1-config \
+        --with-apr-util=/usr/bin/apu-1-config \
+        --with-pcre=/usr \
+        --enable-mods-shared=all \
+        --enable-mpms-shared=all \
+        --with-mpm=prefork \
+        --enable-ssl \
+        --with-ssl \
+        --enable-proxy \
+        --enable-cache \
+        --enable-disk-cache \
+        --enable-mem-cache \
+        --enable-file-cache \
+        --enable-ldap \
+        --enable-authnz-ldap \
+        --enable-cgid \
+        --enable-cgi \
+        --enable-authn-anon \
+        --enable-authn-alias \
+        --disable-imagemap \
+        --enable-proxy-connect \
+        --enable-proxy-http \
+        --enable-proxy-ftp \
+        --enable-deflate \
+        --enable-dbd \
+        --enable-exception-hook \
+        --enable-dav \
+        --enable-dav-fs \
+        --enable-dav-lock
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    description: "Apache HTTP Server (development headers)"
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - perl
+        - apr-util-dev
+
+  - name: ${{package.name}}-doc
+    description: "Apache HTTP Server (documentation)"
+    pipeline:
+      - uses: split/manpages
+      - runs: |
+          set -x
+          mkdir -p "${{targets.subpkgdir}}"/usr/share/doc/apache2
+          mv "${{targets.destdir}}"/usr/share/apache2/default-site/htdocs/manual "${{targets.subpkgdir}}"/usr/share/doc/apache2/
+          rm -rf /usr/share/apache2/default-site/htdocs
+
+  - name: ${{package.name}}-utils
+    description: "Apache HTTP Server (utilities)"
+    pipeline:
+      - runs: |
+          set -x
+          # Move utilities to usr/bin
+          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          for i in ab htdbm htdigest htpasswd httxt2dbm logresolve; do
+            mv "${{targets.destdir}}"/usr/bin/$i "${{targets.subpkgdir}}"/usr/bin/
+          done
+          # Move utilities to usr/sbin
+          mkdir -p "${{targets.subpkgdir}}"/usr/sbin
+          for i in checkgid fcgistarter htcacheclean rotatelogs; do
+            mv "${{targets.destdir}}"/usr/sbin/$i "${{targets.subpkgdir}}"/usr/sbin/
+          done
+
+  - name: ${{package.name}}-data
+    description: "Apache HTTP Server (data files)"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/share/apache2
+          mv "${{targets.destdir}}"/usr/share/apache2/icons "${{targets.subpkgdir}}"/usr/share/apache2/
+          mv "${{targets.destdir}}"/usr/share/apache2/error "${{targets.subpkgdir}}"/usr/share/apache2/
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - runs: |
+        #!/bin/bash
```

Comment:
This can be omitted as it does not set the shell for the pipeline, unless running in bash was intentional (from where you'd use `bash -c ...`)